robot/Cumulus/tests/browser/engagement_plans/add_engagement_plan_to_contact.robot

```diff
@@ -2,34 +2,72 @@
-    Set Suite Variable    ${plan_name}
-    &{contact} =  API Create Contact    MailingStreet=50 Fremont Street    MailingCity=San Francisco    MailingPostalCode=95320    MailingState=CA    MailingCountry=USA
-    Store Session Record    Account    &{contact}[AccountId]
-    Go To Record Home  &{contact}[Id]
-    Select Tab  Related
-    Click Related List Button  Engagement Plans    New
-    Populate Lookup Field    Engagement Plan Template    ${plan_name}
-    Click Modal Button        Save
-    Validate Related Record Count    Engagement Plans   1
-
-Delete Engagement Plan
-    [tags]  unstable
-    ${plan_num}    Verify Eng Plan Exists    ${plan_name}
-    Click Related Item Popup Link    Engagement Plans    ${plan_num}    Delete
-    Click Modal Button        Delete
-    Verify Occurence          Engagement Plans       0
-    
-Verify Tasks Exist Under Activity
-    [tags]  unstable
-    Scroll Page To Location    0    0
-    Click More Activity Button    
-    Check Activity Tasks    ${task1}    ${sub_task}    ${task2}
+    [Documentation]                      Create an Engagement plan Template with two tasks and one subtask. And  a contact
+    ...                                  Link the  engagement plan for the contact and verify
+    ...                                  There is one engagement plan set up for the contact
+    [tags]                               W-038641                 feature:Engagements
+
+
+    Go To Page                                       Listing                            Engagement_Plan_Template__c
+    Go To Engagement Plan Page                       create
+    Wait For Locator                                 frame                              Manage Engagement Plan Template
+
+    Select Frame And Click Element                   Manage Engagement Plan Template    id    idName
+    Enter Eng Plan Values                            idName                             Automation_Plan
+    Enter Eng Plan Values                            idDesc                             This plan is created via Automation
+    Click Button                                     Add Task
+    Wait Until Page Contains                         Task 1
+    Enter Task Id and Subject                        Task 1                             Task_1
+    Click Task Button                                  1                                Add Dependent Task
+    Enter Task Id and Subject                        Task 1-1                           Sub_task_1.1
+    Click Button                                     Add Task
+    Wait Until Page Contains                         Task 2
+    Enter Task Id and Subject                        Task 2                             Task_2
+    Page Scroll To Locator                           button                             Save
+    Click Button                                     Save
+    wait until location contains                     /view
```

Comment:
I suggest adding current page should be Details...  step here instead of this

Reply:
Done